### PR TITLE
Work on the forgot passward page

### PR DESCRIPTION
**Pull request by jaypal412**
## Why?

**Feature:**   
1. Work on forgot a password.
2. Work on the Update password page.
3. Work on the profile page 
4. Work on the checkout flow

## This change addresses the need by:
These changes are  done using bootstrap and HTML

[delivers #158683303]
[Story](https://www.pivotaltracker.com/story/show/158683303)

### CODE REVIEW

File: src/app/checkout/payment/payment.component.html [hunk 1]
```diff
@@ -1,40 +1,44 @@
...
+					<app-payment-modes-list [orderNumber]="orderNumber$ | async" [paymentAmount]="totalCartValue$ | async" [address]="address$ | async"></app-payment-modes-list>
+				</div>
 			</div>
-			<div class="shipping">
-				<span>Delivery</span>
-				<span class="shipping-fee c-green">FREE</span>
+			<div class="right col-12 col-md-4 col-lg-4 m-0">
+				<div class="mini-bag-summary">
+					<div class="lbl">ORDER SUMMARY</div>
+					<div class="items">{{totalCartItems$ | async}} ITEMS</div>
```
Comment (gopalshimpi) [inline src/app/checkout/payment/payment.component.html]: please take care of indentation

File: src/app/checkout/order-success/order-success.component.html [hunk 2]
```diff
@@ -1,67 +1,77 @@
 <div class="container" *ngIf="orderDetails">
-  <div class=col-12>
-    <h1>Thank you for your order!</h1>
+  <div class="col-12 thankyou shadow-sm mt-5 mb-3">
+    <h1>thank you for your order!</h1>
```
Comment (gopalshimpi) [inline src/app/checkout/order-success/order-success.component.html]: spelling of `Thank you`

File: src/app/checkout/order-success/order-success.component.html [hunk 3]
```diff
@@ -1,67 +1,77 @@
...
     <div class="float-left">
-      <i class="fa fa-check-circle fa-3x text-success"></i>
+      <i class="fa fa-check-circle text-success"></i>
     </div>
     <div>
-      <p class="w-75 text-success">Your order has been placed and is being processed. When the item(s) are shipped,you will receive an email with the
+      <p class="w-75 text-success">Your order has been placed and is being processed. When tde item(s) are shipped,you will receive an email witd tde
```
Comment (gopalshimpi) [inline src/app/checkout/order-success/order-success.component.html]: spelling mistake `the`, `with`

File: src/app/checkout/payment/payment.component.html (hunk 1 above)
Comment (jaypal412) [reply src/app/checkout/payment/payment.component.html]: Done

File: src/app/checkout/order-success/order-success.component.html (hunk 2 above)
Comment (jaypal412) [reply src/app/checkout/order-success/order-success.component.html]: Done, But it is handled using the CSS already

File: src/app/checkout/order-success/order-success.component.html (hunk 3 above)
Comment (jaypal412) [reply src/app/checkout/order-success/order-success.component.html]: Done

File: src/app/user/components/orders/orders.component.html [hunk 4]
```diff
@@ -1,13 +1,16 @@
...
-  </div>
-</div>
 
-<ng-template #elseBlock>
-  <div class="cart-empty">
-    <div class="empty-cart-icon"></div>
-    <div class="empty-cart-message">You have'nt ordered anything yet.</div>
-    <button type="button" class="btn btn-primary" [routerLink]="['/']">Start Shopping</button>
+<div class="container">
+  <div *ngIf="(orders$ | async).length; else elseBlock">
```
Comment (pkrawat1) [inline src/app/user/components/orders/orders.component.html]: why multiple async here?

**Pull request merged**